mcs/tools/nunit-lite/Makefile

```diff
@@ -0,0 +1,4 @@
+thisdir = tools/nunit-lite
```

Comment:
should we move this one level up so that it sits next to `nunit24`?

I suppose we don't want to ship nunit-lite with Mono (as opposed to most stuff in tools).